src/apps/HeyGPT/HeyGPT/App.xaml.cs

```diff
@@ -20,17 +19,79 @@
 
 namespace HeyGPT.App;
 
-// To learn more about WinUI 3, see https://docs.microsoft.com/windows/apps/winui/winui3/.
 public partial class App : Application
 {
-    // The .NET Default Host provides dependency injection, configuration, logging, and other services.
-    // https://docs.microsoft.com/dotnet/core/extensions/generic-host
-    // https://docs.microsoft.com/dotnet/core/extensions/dependency-injection
-    // https://docs.microsoft.com/dotnet/core/extensions/configuration
-    // https://docs.microsoft.com/dotnet/core/extensions/logging
-    public IHost Host
+    public IHost Host { get; }
+
+    // TODO: We probably want to manage the MainWindow in a dedicated service.
```

Comment:
There is a lot of work to do to make this a solid application... Since Rome is not built in a day we add bread crumbs to work on later.